channels/channel.nos/nos2010/chn_nos2010.py

```diff
@@ -1331,6 +1332,81 @@ def update_video_item_live(self, item):
         item.complete = True
         return item
 
+    def create_iptv_streams(self, parameter_parser):
+        """ Fetch the available live channels using EPG endpoint and format them into JSON-STREAMS
+
+        :param ActionParser parameter_parser: a ActionParser object to is used to parse and
+                                                   create urls
+
+        :return: Formatted stations
+        :rtype: list
+        """
+
+        epg_data = JsonHelper.loads(UriHandler.open(datetime.datetime.now().strftime("https://start-api.npo.nl/epg/%Y-%m-%d?type=tv"),
```

Comment:
Don't use the `JsonHelper(<data>)`, not the `loads`. That will give you a `JsonHelper` object with a `get_value` method to safely get stuff.

Besides this: I don't like chaining calls like this. I would prefer it to be 2 lines: 1 with the fetching of data, one with processing. Also makes debugging easier.

Reply:
Makes sense, thanks!